packages/storybook/stories/buttons.stories.mdx

```diff
@@ -20,26 +21,14 @@ import { Meta, Canvas } from '@storybook/addon-docs/blocks';
   </Story>
 </Canvas>
 
-## Primary buttons
-
-<Canvas>
-  <Story name="Primary">
-    <div>
-      <button type="button" class="usa-button-primary va-button-primary">
-        Button
-      </button>
-    </div>
-  </Story>
-</Canvas>
-
 ## Secondary buttons
 
 <Canvas>
   <Story name="Secondary">
-    <div>
-      <button class="usa-button-secondary">Button</button>
-      <button class="usa-button-secondary usa-button-active">Active</button>
-      <button class="usa-button-secondary usa-button-hover">Hover</button>
+    <div name="Secondary Buttons">
```

Comment:
Adding the `name` here on the div might not be necessary. In the sass sample I posted I had the `[name]` selector because it looked like storybook was adding that already.

![`name="Secondary"` attribute in markup for targeting](https://user-images.githubusercontent.com/2008881/168169953-4eb401db-5dd3-4c8b-a566-a497e24eb97c.png)

Reply:
This was required to get the styling to work properly on design.va.gov. Without it, it renders a div without a name and the styling is never applied.